code/src/Icons.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './icons.css';
+
+const Icons = () => {
+  return (
+    <>
```

Comment:
If your component returns only one root element, there is no need to wrap it around React Fragment.

In this case WE NEED to use React Fragment :

```jsx
return (
    <div>My element 1</div>
    <div>My element 2</div>
);
```

so, after necessary changes it looks like this :

```jsx
return (
    <>
        <div>My element 1</div>
	<div>My element 2</div>
    </>
);
```

But in this case WE DON'T NEED to use React Fragment :

```jsx
return (
    <div>My element 1</div>
);
```